index.php

```diff
@@ -4,6 +4,16 @@
 
 Configuration::verifyInstallation();
 Configuration::loadConfiguration();
+
+date_default_timezone_set(Configuration::getConfig('system', 'timezone'));
+
+if (Configuration::getConfig('proxy', 'url')) {
+    define('PROXY_URL', Configuration::getConfig('proxy', 'url'));
+}
+define('PROXY_BYBRIDGE', Configuration::getConfig('proxy', 'by_bridge'));
+define('PROXY_NAME', Configuration::getConfig('proxy', 'name'));
+define('CUSTOM_CACHE_TIMEOUT', Configuration::getConfig('cache', 'custom_timeout'));
```

Comment:
The refactoring commit looks otherwise okay but why do we even set these constants in the first place, rather than just getting the values from `Configuration` object (that can potentially by passed via dependency injection).

Reply:
You are right. These are legacy consts which can be inlined but I didn't feel like doing it because the logic is convoluted.

Reply:
Done that in https://github.com/RSS-Bridge/rss-bridge/pull/2938.